structure_tests/structure_test_v1.go

```diff
@@ -29,6 +29,7 @@ type StructureTestv1 struct {
 	CommandTests       []CommandTestv1
 	FileExistenceTests []FileExistenceTestv1
 	FileContentTests   []FileContentTestv1
+	LicenseTests       []string
```

Comment:
Then debian could just be a bool, files could be a list of strings.

license_test:
  debian: true
  files: [/app/mypackage/LICENSE, /foo/otherthing/LICENSE.txt]